q2_intervention/_utilities.py

```diff
@@ -27,28 +27,40 @@
 TEMPLATES = pkg_resources.resource_filename('q2_intervention', 'assets')
 
 
+def _check_inputs(state_1, state_2):
```

Comment:
I intend to validate more inputs in a future PR so will use `_validate_input_values` instead of `_validate_state_values`